apps/web/components/v2/eventtype/EventAdvancedTab.tsx

```diff
@@ -329,9 +326,10 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupInfered
                     label={t("number_of_seats")}
                     type="number"
                     defaultValue={value || 2}
+                    min={1}
                     addOnSuffix={<>{t("seats")}</>}
                     onChange={(e) => {
-                      onChange(Number(e.target.value));
+                      onChange(Math.abs(Number(e.target.value)));
```

Comment:
NIT improvement to don't allow negative numbers on input field. (Should validate on form and server too probably)